bindings/xmtp_rust_swift/Generated/xmtp_rust_swift/xmtp_rust_swift.swift

```diff
@@ -1,3 +1,6 @@
+public func create_envelope<GenericIntoRustString: IntoRustString>(_ topic: GenericIntoRustString, _ sender_time_ns: UInt64, _ payload: RustVec<UInt8>) -> Envelope {
```

Comment:
Think you might need to merge main into your branch to get the .gitignore changes